### PR TITLE
CDAP-811: Delete app-manager scripts in favor of CLI

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: MovieRecommender/README.rst [hunk 1]
```diff
@@ -43,32 +43,41 @@ Installation & Usage
...
+ From the Standalone CDAP SDK directory, use the Command-line Interface::
 
-  bin/app-manager.sh --host [host] --action start
+  $ ./bin/cdap-cli.sh start flow MovieRecommender.RatingsFlow
+  $ ./bin/cdap-cli.sh start service MovieRecommender.MovieRecommenderService
+  $ ./bin/cdap-cli.sh start service MovieRecommender.MovieDictionaryService
   
-Make sure that the Services and Procedures are running (note that the
-``RecommendationBuilder`` Spark program will be started later)::
+Make sure that the Flows and Services are running (note that the
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: Check that the Flow and Services [note that there is only one flow...]

File: MovieRecommender/README.rst [hunk 2]
```diff
@@ -43,32 +43,41 @@ Installation & Usage
...
 
-  bin/app-manager.sh --host [host] --action deploy
+Deploy the Application to a CDAP instance:
+ From the Standalone CDAP SDK directory, use the Command-line Interface::
+
+  $ ./bin/cdap-cli.sh deploy app <path-to-MovieRecommender-jar-file>
   
-Start the Application Services and Procedures::
+Start the Application Flows and Services:
+ From the Standalone CDAP SDK directory, use the Command-line Interface::
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: Command Line Interface [common; we've decided to drop the hyphen...]

File: MovieRecommender/README.rst [hunk 3]
```diff
@@ -43,32 +43,41 @@ Installation & Usage
...
   bin/ingest-data.sh --host [host]
 
-Run the ``RecommendationBuilder`` Spark Program::
+Run the ``RecommendationBuilder`` Spark Program:
+ From the Standalone CDAP SDK directory, use the Command-line Interface::
 
-  bin/app-manager.sh --host [host] --action run
+  $ ./bin/cdap-cli.sh start spark MovieRecommender.RecommendationBuilder
 
 The Spark program may take a couple of minutes to complete. You can check if it is complete by its
```
Comment (awholegunch) [inline MovieRecommender/README.rst]:
```
You can check if it has completed by checking its status (once done, it becomes ``STOPPED``)::
```

File: MovieRecommender/README.rst [hunk 4]
```diff
@@ -86,9 +95,12 @@ This will return a JSON response of rated and recommended movies::
 
 Alternately, you can use the CDAP Console Procedure page to execute the queries above.
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: Didn't these procedures get dropped? Lines 84 onwards probably need updating, right?

**Comment by awholegunch** [comment]
There are references to Procedures in the instructions for MovieRecommener that require replacement with either curl or CCLI commands (or both; we should one of each, ideally).

File: MovieRecommender/README.rst [hunk 5]
```diff
@@ -43,52 +43,63 @@ Installation & Usage
...
   libexec\curl -v -d '{"userId":"1"}' \
-  -X POST 'http://localhost:10000/v2/apps/MovieRecommender/procedures/RecommendMovieProcedure/methods/getRecommendation'
+  -X POST 'http://localhost:10000/v2/apps/MovieRecommender/services/MovieRecommenderService/methods/getRecommendation'
   
 This will return a JSON response of rated and recommended movies::
 
   {"rated":["ratedMovie1","ratedMovie1"],"recommended":["recommendedMovie1","recommendedMovie2"]}
 
-Alternately, you can use the CDAP Console Procedure page to execute the queries above.
+Alternately, you can use the CDAP Console Service page to execute the queries above.
```
Comment (awholegunch) [inline MovieRecommender/README.rst]: Please test this!

```
Alternately, you can use the Command Line Interface to execute this query::

$ ./bin/cdap-cli.sh call service MovieRecommender.MovieRecommenderService POST getRecommendation
```

File: Netlens/README.rst [hunk 6]
```diff
@@ -77,17 +77,26 @@ Build the Application jar::
...
 
-Deploy the Application to a CDAP instance defined by its host (defaults to localhost)::
+Deploy the Application to a CDAP instance:
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-  bin/app-manager.sh --host [host] --action deploy
+  $ ./bin/cdap-cli.sh deploy app <path-to-Netlens-jar-file>
 
-Start the Application Flows and Services::
+Start the Application Flows and Services:
```
Comment (awholegunch) [inline Netlens/README.rst]: Start the Application Flow and Services:

File: Netlens/README.rst [hunk 7]
```diff
@@ -77,17 +77,26 @@ Build the Application jar::
...
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-  bin/app-manager.sh --host [host] --action start
+  $ ./bin/cdap-cli.sh start flow Netlens.AnalyticsFlow
+  $ ./bin/cdap-cli.sh start service Netlens.AnomaliesCountService
+  $ ./bin/cdap-cli.sh start service Netlens.AnomaliesService
+  $ ./bin/cdap-cli.sh start service Netlens.CountersService
 
-Make sure they are running::
+Check they are running:
```
Comment (awholegunch) [inline Netlens/README.rst]: Check that they are running:

File: TwitterSentiment/README.rst [hunk 8]
```diff
@@ -61,17 +61,22 @@ Build the Application jar::
...
 
-Deploy the Application to a CDAP instance defined by its host (defaults to localhost)::
+Deploy the Application to a CDAP instance:
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-  bin/app-manager.sh --host [host] --action deploy
+  $ ./bin/cdap-cli.sh deploy app <path-to-TwitterSentiment-jar-file>
 
-Start Application Flows and Procedures::
+Start Application Flows and Services:
```
Comment (awholegunch) [inline TwitterSentiment/README.rst]: Start Application Flow and Service:

File: TwitterSentiment/README.rst [hunk 9]
```diff
@@ -61,17 +61,22 @@ Build the Application jar::
...
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
 
-  bin/app-manager.sh --host [host] --action start
+  $ ./bin/cdap-cli.sh start flow TwitterSentiment.TwitterSentimentAnalysis
+  $ ./bin/cdap-cli.sh start service TwitterSentiment.SentimentQuery
 
-Make sure they are running::
+Check they are running:
```
Comment (awholegunch) [inline TwitterSentiment/README.rst]: Check that they are running:

File: Wise/README.rst [hunk 10]
```diff
@@ -38,17 +38,27 @@ Build the application by executing::
...
-  $ bin/app-manager.sh --action deploy
-  $ bin/app-manager.sh --action start
-  $ bin/app-manager.sh --action status
-
-On Windows, run::
-
-  $ bin\app-manager.bat deploy
-  $ bin\app-manager.bat start
-  $ bin\app-manager.bat status
+To deploy and start the application, check CDAP is running and then execute:
```
Comment (awholegunch) [inline Wise/README.rst]: check that CDAP is running and then execute:

File: Wise/README.rst [hunk 11]
```diff
@@ -38,17 +38,27 @@ Build the application by executing::
...
+  $ ./bin/cdap-cli.sh start workflow Wise.WiseWorkflow
+  $ ./bin/cdap-cli.sh start service Wise.WiseService
+  $ ./bin/cdap-cli.sh get flow status Wise.WiseFlow
+  $ ./bin/cdap-cli.sh get workflow status Wise.WiseWorkflow
+  $ ./bin/cdap-cli.sh get service status Wise.WiseService
+
+On Windows, run:
+ From the Standalone CDAP SDK directory, use the Command Line Interface::
+
+  > bin\cdap-cli.sh deploy app <path-to-Wise-jar-file>
```
Comment (awholegunch) [inline Wise/README.rst]: cdap-cli.bat [common for Windows examples]

**Comment by awholegunch** [comment]
Our internet was down today, which is why I could not get to these earlier. Let me know when they are ready to re-review.

File: MovieRecommender/README.rst (hunk 5 above)
Comment (ghost) [inline MovieRecommender/README.rst]: must be something like   $ ./bin/cdap-cli.sh call service MovieRecommender.MovieRecommenderService GET recommend/{userId}

**Comment by ghost** [comment]
@awholegunch PR ready for re-review

**Pull request closed without merging**